Documentation/DSCResources/VMHostVss/VMHostVss.md

```diff
@@ -10,12 +10,15 @@
 | **Ensure** | Mandatory | Ensure | Value indicating if the VSS should be Present or Absent. | Present, Absent |
 | **VssName** | Mandatory | string | The name of the VSS. ||
 | **Mtu** | Optional | int | The maximum transmission unit (MTU) associated with this virtual switch in bytes. ||
-| **NumPorts** | Optional | int | The number of ports that this virtual switch is configured to use. ||
 
 ## Description
 
 The resource is used to configure the basic properties of a Virtual Switch (VSS) on an ESXi node.
 
+## Notes
```

Comment:
1. Remove the NumPorts property from the first example as it is no longer used.
2. Add empty line at the end of the file.

Reply:
Changed